src/plugins/console/public/src/es.js

```diff
@@ -32,7 +32,7 @@ module.exports.send = function (method, path, data, server, disable_auth_alert)
   var settings = require("./settings");
 
   var options = {
-    url: '../api/sense/proxy?uri=' + encodeURIComponent(path),
+    url: '../api/console/proxy?uri=' + encodeURIComponent(path),
```

Comment:
Yep, the `../` takes care of traversing out of the `/app` sub directory in `/app/sense` since `sense` is seen as a "filename" and the server enforces the lack of trailing slash.